J/Julia/build_tarballs.jl

```diff
@@ -1,103 +1,50 @@
-    "bfe9df6c52164c90b752cf6f167f69dffb5a0332658d05b0a42bfe18dbdf5e6a",
-]
-
-# Bash recipe for building across all platforms
-script = raw"""
-cd $WORKSPACE/srcdir/julia*/
-
-BUILD_FLAGS=(USECCACHE=1)
-
-# Mount pts for Julia bootstrap
-mount -t devpts -o newinstance jrunpts /dev/pts
-mount -o bind /dev/pts/ptmx /dev/ptmx
-
-# Set prefix-related paths
-#BUILD_FLAGS+=("prefix=${prefix}")
-
-# Set arch-related flags
-case ${target} in
-    x86_64-*)
-        BUILD_FLAGS+=("MARCH=x86-64" "JULIA_CPU_TARGET=generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)")
-    ;;
-    i686-*)
-        BUILD_FLAGS+=("MARCH=pentium4" "JULIA_CPU_TARGET=pentium4;sandybridge,-xsaveopt,clone_all")
-    ;;
-    arm-*)
-        BUILD_FLAGS+=("MARCH=armv7-a" "JULIA_CPU_TARGET=armv7-a;armv7-a,neon;armv7-a,neon,vfp4")
-    ;;
-    ppc64le-*)
-        BUILD_FLAGS+=("JULIA_CPU_TARGET=pwr8")
-    ;;
-    aarch64-*)
-        BUILD_FLAGS+=("MARCH=armv8-a" "JULIA_CPU_TARGET=generic")
-    ;;
-esac
-
-# If we're compiling for Windows, then set XC_HOST
-if [[ ${target} == *mingw* ]]; then
-    BUILD_FLAGS+=("XC_HOST=${target}")
-else
-    export LDFLAGS="$LDFLAGS -Wl,-rpath,${prefix}/lib"
-fi
-
-# Make use of many prebuilt things
-#for proj in BLAS LLVM PCRE MBEDTLS LIBSSH2 CURL LIBGIT2 GMP MPFR; do
-#    BUILD_FLAGS+=(USE_SYSTEM_${proj}=1)
-#done
-
-#if [[ ${nbits} == 64 ]]; then
-#    BUILD_FLAGS+=(LIBBLASNAME=libopenblas LIBBLAS=-lopenblas64_)
-#else
-#    BUILD_FLAGS+=(LIBBLASNAME=libopenblas LIBBLAS=-lopenblas)
-#fi
-
-#BUILD_FLAGS+=(LDFLAGS="$LDFLAGS")
-BUILD_FLAGS+=(LIBSSH2_ENABLE_TESTS=0)
-
-make ${BUILD_FLAGS[@]} -j${nproc} -C deps install-mbedtls
-make ${BUILD_FLAGS[@]} -j${nproc} -C deps install-libgit2
-make ${BUILD_FLAGS[@]} -j${nproc}
-make ${BUILD_FLAGS[@]} install
-"""
+version = v"1.4.1"
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
-    Linux(:x86_64; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Linux(:i686; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Windows(:x86_64; compiler_abi=CompilerABI(:gcc7)),
-    Windows(:i686; compiler_abi=CompilerABI(:gcc7)),
+    Linux(:armv7l; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    Linux(:aarch64; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    Linux(:x86_64; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    Linux(:i686; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    MacOS(:x86_64; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    Windows(:x86_64; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
+    Windows(:i686; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
```

Comment:
FreeBSD too? :wink:

Reply:
Looks like it was added as a source but not a platform.

Reply:
Oh no!  [This commit message](https://github.com/JuliaPackaging/Yggdrasil/pull/1048/commits/4cd3ba16336a98214eb0b2e39b9ab1c17c9b6712) sounds more sarcastic now :sweat_smile: